### PR TITLE
Toolbar Multiline Mode: Update demo after review

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: MVCDemos/Views/Toolbar/Adaptability.cshtml [hunk 1]
```diff
@@ -324,23 +354,31 @@
...
+        DevExpress.ui.notify('The "Code Block" button was clicked');
     }
 
-    function onCheckBoxValueChanged() {
-        DevExpress.ui.notify('The "Navigation Pane" checkbox value was changed');
+    function onQuoteButtonClick() {
+        DevExpress.ui.notify('The "Blockquote" button was clicked');
+    }
+
+    function attachButtonClick() {
```
Comment (ksercs) [inline MVCDemos/Views/Toolbar/Adaptability.cshtml]: Please, ensure that naming is consistent everywhere

```suggestion
    function onAttachButtonClick() {
```

File: JSDemos/Demos/Toolbar/Adaptability/Vue/App.vue [hunk 2]
```diff
@@ -437,6 +464,10 @@ export default {
   margin-bottom: 5px;
 }
 
+.dx-texteditor {
```
Comment (ksercs) [inline JSDemos/Demos/Toolbar/Adaptability/Vue/App.vue:467]: I don't see this change in another frameworks. Is it necessary?

File: MVCDemos/Views/Toolbar/Adaptability.cshtml (hunk 1 above)
Comment (marker-dao) [reply MVCDemos/Views/Toolbar/Adaptability.cshtml]: https://github.com/DevExpress/devextreme-demos/pull/2135/commits/86fbd66c983db71118dea44d78073a2f93d9975c

File: JSDemos/Demos/Toolbar/Adaptability/Vue/App.vue (hunk 2 above)
Comment (marker-dao) [reply JSDemos/Demos/Toolbar/Adaptability/Vue/App.vue:467]: Created task of fix vue-wrapper

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**